File: sdkwork-app-sdk-go/types/activity_detail_vo.go
package types

// 活动详情
type ActivityDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	Title string `json:"title"`
	Description string `json:"description"`
	Type string `json:"type"`
	CoverImage string `json:"coverImage"`
	BannerImages []string `json:"bannerImages"`
	Rules string `json:"rules"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
	ParticipantCount int `json:"participantCount"`
	HasJoined bool `json:"hasJoined"`
	MaxJoinCount int `json:"maxJoinCount"`
	JoinedCount int `json:"joinedCount"`
	RemainingCount int `json:"remainingCount"`
	Rewards []ActivityReward `json:"rewards"`
	ExtraConfig map[string]interface{} `json:"extraConfig"`
}

File: sdkwork-app-sdk-csharp/Models/ActivityDetailVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ActivityDetailVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public string? Type { get; set; }
        public string? CoverImage { get; set; }
        public List<string>? BannerImages { get; set; }
        public string? Rules { get; set; }
        public string? StartTime { get; set; }
        public string? EndTime { get; set; }
        public int? ParticipantCount { get; set; }
        public bool? HasJoined { get; set; }
        public int? MaxJoinCount { get; set; }
        public int? JoinedCount { get; set; }
        public int? RemainingCount { get; set; }
        public List<ActivityReward>? Rewards { get; set; }
        public Dictionary<string, object>? ExtraConfig { get; set; }
    }
}
